.golangci.yml

```diff
@@ -27,12 +27,14 @@ linters-settings:
     settings:
       printf:
         funcs:
-          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Infof
-          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Warnf
-          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Errorf
-          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
+          - (github.com/hyperledger/aries-framework-go/pkg/common/log.Log).Debugf
+          - (github.com/hyperledger/aries-framework-go/pkg/common/log.Log).Infof
+          - (github.com/hyperledger/aries-framework-go/pkg/common/log.Log).Warnf
+          - (github.com/hyperledger/aries-framework-go/pkg/common/log.Log).Errorf
+          - (github.com/hyperledger/aries-framework-go/pkg/common/log.Log).Fatalf
+          - (github.com/hyperledger/aries-framework-go/pkg/common/log.Log).Panicf
   golint:
-    min-confidence: 0.8
+    min-confidence: 0.6
```

Comment:
0.8 was missing an obvious lint issue so I bumped to 0.6 (3/5ths)

Reply:
Note if I set to 0 then we get a few more linter warnings that I haven't reviewed yet.